sysutils/ntpsec/Portfile

```diff
@@ -42,15 +50,14 @@ foreach s ${pythons_suffixes} {
     variant ${p} description "Build ${name} to use Python ${v}" conflicts {*}${c} ""
 }
 
-# Default to +python27
+# Default to +python312
 #
-# Since this port provides Python modules that may be used by users, changing
-# the Python version may break existing user code.  Since earlier versions of
-# this port had no pythonXX variant, there is no established default variant
-# to avoid such a switch on an upgrade.  Hence, the default should remain at
-# +python27 for a while.
+# This is an "almost latest" version of Python, without the "bleeding edge"
+# issues of python313, including its current inability to build on all
+# platforms.  It can be requested explicitly, however.  Upgrades to existing
+# installs will preserve the current selection.
 #
-set pyver_no_dot "27"
+set pyver_no_dot "312"
```

Comment:
there is nothin "bleeding edge" about Python 3.13 and since that is the default version it should be used, following MacPorts' Python policy. If you want to support very old OSes that cannot build it that's fine - but then specify that only those versions use an older Python.